### PR TITLE
Added VIA support to Efreet

### PR DESCRIPTION
**Pull request by Wilba6582**
## Description

Added VIA support to Efreet, a keyboard designed by Soran.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
All the errors in Travis are with Community keymaps and not relevant to these changes.

File: keyboards/efreet/rules.mk [hunk 1]
```diff
@@ -14,11 +14,11 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/efreet/rules.mk]: While you're here, would you mind removing these:
```suggestion
BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
```
And the others that GitHub won't let me add suggestions to as it's outside the hunk.

File: keyboards/efreet/rules.mk [hunk 2]
```diff
@@ -14,11 +14,11 @@ BOOTLOADER = atmel-dfu
...
 MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
```
Comment (fauxpark) [inline keyboards/efreet/rules.mk]:
```suggestion
MOUSEKEY_ENABLE = yes       # Mouse keys
```

File: keyboards/efreet/rules.mk [hunk 3]
```diff
@@ -14,11 +14,11 @@ BOOTLOADER = atmel-dfu
...
 EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
```
Comment (fauxpark) [inline keyboards/efreet/rules.mk]:
```suggestion
EXTRAKEY_ENABLE = yes       # Audio control and System control
```

File: keyboards/efreet/rules.mk [hunk 4]
```diff
@@ -14,11 +14,11 @@ BOOTLOADER = atmel-dfu
...
 CONSOLE_ENABLE = no        	# Console for debug(+400)
```
Comment (fauxpark) [inline keyboards/efreet/rules.mk]:
```suggestion
CONSOLE_ENABLE = no        	# Console for debug
```

File: keyboards/efreet/rules.mk (hunk 1 above)
Comment (Wilba6582) [reply keyboards/efreet/rules.mk]: Fixed

**Pull request merged**